TESTS/psa/attestation/main.cpp

```diff
@@ -128,11 +129,13 @@ utest::v1::status_t case_teardown_handler(const Case *const source, const size_t
 
 utest::v1::status_t case_setup_handler(const Case *const source, const size_t index_of_case)
 {
+    psa_status_t status = mbed_psa_reboot_and_request_new_security_state(PSA_LIFECYCLE_ASSEMBLY_AND_TEST);
```

Comment:
Need to choose the correct way to clean this test,
in tear down the test uses 'psa_destroy_key(handle);' and in init the test uses mbed_psa_reboot_and_request_new_security_state.

it seems the correct way is or both use 'mbed_psa_reboot_and_request_new_security_state' or both uses 'psa_destroy_key'

in addition, does the 'mbed_psa_reboot_and_request_new_security_state' will work if using protected storage and not secure internal storage?

Reply:
@netanelgonen 
what does PS has to do with attestation? please elaborate

Reply:
@netanelgonen, despite its name, the `mbed_psa_reboot_and_request_new_security_state()`API will not reboot the system when moving from `PSA_LIFECYCLE_ASSEMBLY_AND_TEST` to `PSA_LIFECYCLE_ASSEMBLY_AND_TEST`. The only effect it will have is to clean PSA RoT state. (currently just wiping out ITS).

Reply:
@orenc17 @alzix, please choose 'mbed_psa_reboot_and_request_new_security_state' or both uses 'psa_destroy_key'